### ISSUE TITLE
Version 2 of "I installed ipykernel 6, but I still can't use run-by-line"

### ISSUE THREAD

**Issue by greazer** [microsoft/vscode-jupyter#7128]
When I entered bug #7127, I thought that restarting the kernel should allow me to use run-by-line. However, after restarting the kernel once, then twice it still doesn't work. I'm still told I need to install ipykernel.

I think it may be installing to the wrong environment.

This is 
Version: 1.60.0-insider (user setup)
Commit: 58454f7bec838f79fe2422ab5914419147d82e40
Date: 2021-08-13T05:15:29.421Z
Electron: 13.1.8
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Windows_NT x64 10.0.22000

Jupyter: v2021.8.2027072902

Here's terminal output from the second time I tried having VS Code install it for me: 

conda install --name base --update-deps --force-reinstall ipykernel -y
Collecting package metadata (current_repodata.json): done
Solving environment: \ Solving environment/ done
done

## Package Plan ##

  environment location: C:\Users\jimg\anaconda3

  added / updated specs:
    - backcall
    - backports
    - backports.functools_lru_cache
    - ca-certificates
    - certifi
    - colorama
    - decorator
    - ipykernel
    - ipython
    - ipython_genutils
    - jedi
    - jupyter_client
    - jupyter_core
    - libsodium
    - openssl
    - parso
    - pickleshare
    - pip
    - prompt-toolkit
    - prompt_toolkit
    - pygments
    - python-dateutil
    - python=3.7
    - python_abi
    - pywin32
    - pyzmq
    - setuptools
    - six
    - sqlite
    - tornado
    - traitlets
    - ucrt
    - vc
    - vs2015_runtime
    - wcwidth
    - wheel
    - wincertstore
    - zeromq

Preparing transaction: done
Verifying transaction: done
Executing transaction: done

Conda List in this environment still shows ipykernel 5.5.5 as being installed:

PS C:\Users\...> conda list
# packages in environment at C:\Users\me\anaconda3:
#
# Name                    Version                   Build  Channel
_anaconda_depends         2020.07                  py37_0
_ipyw_jlab_nb_ext_conf    0.1.0                    py37_0
alabaster                 0.7.12                     py_0    conda-forge
anaconda                  custom                   py37_1
anaconda-client           1.7.2              pyhd8ed1ab_1    conda-forge
anaconda-navigator        1.9.12                   py37_0
anaconda-project          0.10.1             pyhd8ed1ab_0    conda-forge
anyio                     3.3.0            py37h03978a9_0    conda-forge
appdirs                   1.4.4              pyh9f0ad1d_0    conda-forge
argh                      0.26.2          pyh9f0ad1d_1002    conda-forge
argon2-cffi               20.1.0           py37hcc03f2d_2    conda-forge
asn1crypto                1.4.0              pyh9f0ad1d_0    conda-forge
astroid                   2.6.6            py37h03978a9_0    conda-forge
astropy                   4.3.1            py37hcc03f2d_0    conda-forge
async_generator           1.10                       py_0    conda-forge
atomicwrites              1.4.0              pyh9f0ad1d_0    conda-forge
attrs                     21.2.0             pyhd8ed1ab_0    conda-forge
autopep8                  1.5.7              pyhd8ed1ab_0    conda-forge
babel                     2.9.1              pyh44b312d_0    conda-forge
backcall                  0.2.0              pyh9f0ad1d_0    conda-forge
backports                 1.0                        py_2    conda-forge
backports.functools_lru_cache 1.6.4              pyhd8ed1ab_0    conda-forge
backports.shutil_get_terminal_size 1.0.0                      py_3    conda-forge
backports.tempfile        1.0                        py_0    conda-forge
backports.weakref         1.0.post1       py37hc8dfbb8_1002    conda-forge
bcrypt                    3.2.0            py37hcc03f2d_1    conda-forge
beautifulsoup4            4.9.3              pyhb0f4dca_0    conda-forge
bitarray                  2.0.1            py37hcc03f2d_0    conda-forge
bkcharts                  0.2                      py37_0
blas                      1.0                         mkl
bleach                    4.0.0              pyhd8ed1ab_0    conda-forge
blosc                     1.21.0               h0e60522_0    conda-forge
bokeh                     2.3.3            py37h03978a9_0    conda-forge
boto                      2.49.0                     py_0    conda-forge
bottleneck                1.3.2            py37hda49f71_3    conda-forge
branca                    0.4.1                    pypi_0    pypi
brotlipy                  0.7.0           py37hcc03f2d_1001    conda-forge
bzip2                     1.0.8                h8ffe710_4    conda-forge
ca-certificates           2021.5.30            h5b45459_0    conda-forge
cached-property           1.5.2                hd8ed1ab_1    conda-forge
cached_property           1.5.2              pyha770c72_1    conda-forge
certifi                   2020.12.5                pypi_0    pypi
cffi                      1.14.6           py37hd8e9650_0    conda-forge
cfitsio                   3.470                h0af3d06_7    conda-forge
chardet                   4.0.0            py37h03978a9_1    conda-forge
charls                    2.2.0                h39d44d4_0    conda-forge
charset-normalizer        2.0.0              pyhd8ed1ab_0    conda-forge
click                     8.0.1            py37h03978a9_0    conda-forge
cloudpickle               1.6.0                      py_0    conda-forge
clyent                    1.2.2                      py_1    conda-forge
colorama                  0.4.4              pyh9f0ad1d_0    conda-forge
colorcet                  2.0.6              pyhd8ed1ab_0    conda-forge
comtypes                  1.1.10           py37h03978a9_0    conda-forge
conda                     4.10.3           py37h03978a9_0    conda-forge
conda-build               3.18.11                  py37_0
conda-env                 2.6.0                         1
conda-pack                0.6.0              pyhd3deb0d_0    conda-forge
conda-package-handling    1.7.3            py37ha54c9ec_0    conda-forge
conda-verify              3.4.2                      py_1
confuse                   1.3.0                    pypi_0    pypi
console_shortcut          0.1.1                         4
contextlib2               21.6.0             pyhd8ed1ab_0    conda-forge
cryptography              3.4.7            py37h20c650d_0    conda-forge
cudatoolkit               11.1.1               heb2d755_7    conda-forge
curl                      7.78.0               h789b8ee_0    conda-forge
cycler                    0.10.0                     py_2    conda-forge
cython                    0.29.24          py37hf2a7229_0    conda-forge
cytoolz                   0.11.0           py37hcc03f2d_3    conda-forge
dask                      2021.7.2           pyhd8ed1ab_0    conda-forge
dask-core                 2021.7.2           pyhd8ed1ab_0    conda-forge
dataclasses               0.8                pyhc8e2a94_1    conda-forge
decorator                 5.0.9              pyhd8ed1ab_0    conda-forge
defusedxml                0.7.1              pyhd8ed1ab_0    conda-forge
diff-match-patch          20200713           pyh9f0ad1d_0    conda-forge
display                   1.0.0                    pypi_0    pypi
distlib                   0.3.1                    pypi_0    pypi
distributed               2021.7.2         py37h03978a9_0    conda-forge
docutils                  0.17.1           py37h03978a9_0    conda-forge
entrypoints               0.3             pyhd8ed1ab_1003    conda-forge
et_xmlfile                1.0.1                   py_1001    conda-forge
fastcache                 1.1.0            py37hcc03f2d_2    conda-forge
filelock                  3.0.12             pyh9f0ad1d_0    conda-forge
flake8                    3.9.2              pyhd8ed1ab_0    conda-forge
flask                     2.0.1              pyhd8ed1ab_0    conda-forge
folium                    0.11.0                   pypi_0    pypi
freetype                  2.10.4               h546665d_1    conda-forge
fsspec                    2021.7.0           pyhd8ed1ab_0    conda-forge
future                    0.18.2           py37h03978a9_3    conda-forge
get_terminal_size         1.0.0                h38e98db_0
gevent                    21.8.0           py37hcc03f2d_0    conda-forge
giflib                    5.2.1                h8d14728_2    conda-forge
glob2                     0.7                        py_0    conda-forge
gmpy2                     2.1.0b5          py37hc3278d5_0    conda-forge
greenlet                  1.1.1            py37hf2a7229_0    conda-forge
h5py                      3.3.0           nompi_py37he280515_100    conda-forge
hdf5                      1.10.6          nompi_h5268f04_1114    conda-forge
heapdict                  1.0.1                      py_0    conda-forge
holoviews                 1.14.5             pyhd8ed1ab_0    conda-forge
html5lib                  1.1                pyh9f0ad1d_0    conda-forge
htmlmin                   0.1.12                   pypi_0    pypi
hvplot                    0.7.2              pyh6c4a22f_0    conda-forge
icc_rt                    2019.0.0             h0cc432a_1
icu                       68.1                 h0e60522_0    conda-forge
idna                      3.1                pyhd3deb0d_0    conda-forge
imagecodecs               2021.7.30        py37h91eda04_0    conda-forge
imagehash                 4.1.0                    pypi_0    pypi
imageio                   2.9.0                      py_0    conda-forge
imagesize                 1.2.0                      py_0    conda-forge
importlib-metadata        4.6.3            py37h03978a9_0    conda-forge
importlib_metadata        4.6.3                hd8ed1ab_0    conda-forge
iniconfig                 1.1.1              pyh9f0ad1d_0    conda-forge
intel-openmp              2021.3.0          h57928b3_3372    conda-forge
intervaltree              3.0.2                      py_0    conda-forge
ipykernel                 5.5.5            py37h7813e69_0    conda-forge
ipython                   7.12.0           py37h5ca1d4c_0    conda-forge
ipython_genutils          0.2.0                      py_1    conda-forge
ipywidgets                7.6.3              pyhd3deb0d_0    conda-forge
isort                     5.9.3              pyhd8ed1ab_0    conda-forge
itsdangerous              2.0.1              pyhd8ed1ab_0    conda-forge
jdcal                     1.4.1                      py_0    conda-forge
jedi                      0.14.1                   py37_0
jinja2                    3.0.1              pyhd8ed1ab_0    conda-forge
joblib                    1.0.1              pyhd8ed1ab_0    conda-forge
jpeg                      9d                   h8ffe710_0    conda-forge
json5                     0.9.5              pyh9f0ad1d_0    conda-forge
jsonschema                3.2.0              pyhd8ed1ab_3    conda-forge
jupyter                   1.0.0            py37h03978a9_6    conda-forge
jupyter_client            6.1.12             pyhd8ed1ab_0    conda-forge
jupyter_console           6.4.0              pyhd8ed1ab_0    conda-forge
jupyter_core              4.7.1            py37h03978a9_0    conda-forge
jupyter_server            1.10.2             pyhd8ed1ab_0    conda-forge
jupyterlab                3.1.6              pyhd8ed1ab_0    conda-forge
jupyterlab_pygments       0.1.2              pyh9f0ad1d_0    conda-forge
jupyterlab_server         2.7.0              pyhd8ed1ab_0    conda-forge
jupyterlab_widgets        1.0.0              pyhd8ed1ab_1    conda-forge
jxrlib                    1.1                  h8ffe710_2    conda-forge
keyring                   23.0.1           py37h03978a9_0    conda-forge
kiwisolver                1.3.1            py37h8c56517_1    conda-forge
krb5                      1.19.2               hbae68bd_0    conda-forge
lazy-object-proxy         1.6.0            py37hcc03f2d_0    conda-forge
lcms2                     2.12                 h2a16943_0    conda-forge
lerc                      2.2.1                h0e60522_0    conda-forge
libaec                    1.0.5                h39d44d4_0    conda-forge
libarchive                3.5.1                hb45042f_2    conda-forge
libblas                   3.9.0                     8_mkl    conda-forge
libbrotlicommon           1.0.9                h8ffe710_5    conda-forge
libbrotlidec              1.0.9                h8ffe710_5    conda-forge
libbrotlienc              1.0.9                h8ffe710_5    conda-forge
libcblas                  3.9.0                     8_mkl    conda-forge
libclang                  11.1.0          default_h5c34c98_1    conda-forge
libcurl                   7.78.0               h789b8ee_0    conda-forge
libdeflate                1.8                  h8ffe710_0    conda-forge
libiconv                  1.16                 he774522_0    conda-forge
liblapack                 3.9.0                     8_mkl    conda-forge
liblief                   0.11.5               h0e60522_0    conda-forge
libllvm9                  9.0.1                hab3b255_3    conda-forge
libpng                    1.6.37               h1d00b33_2    conda-forge
libsodium                 1.0.18               h8d14728_1    conda-forge
libspatialindex           1.9.3                h39d44d4_4    conda-forge
libssh2                   1.9.0                h680486a_6    conda-forge
libtiff                   4.3.0                h0c97f57_0    conda-forge
libuv                     1.42.0               h8ffe710_0    conda-forge
libwebp-base              1.2.0                h8ffe710_2    conda-forge
libxml2                   2.9.12               hf5bbc77_0    conda-forge
libxslt                   1.1.33               h65864e5_2    conda-forge
libzopfli                 1.0.3                h0e60522_0    conda-forge
llvmlite                  0.36.0           py37habb0c8c_0    conda-forge
locket                    0.2.0                      py_2    conda-forge
lxml                      4.6.3            py37hd07aab1_0    conda-forge
lz4-c                     1.9.3                h8ffe710_1    conda-forge
lzo                       2.10              he774522_1000    conda-forge
m2w64-gcc-libgfortran     5.3.0                         6    conda-forge
m2w64-gcc-libs            5.3.0                         7    conda-forge
m2w64-gcc-libs-core       5.3.0                         7    conda-forge
m2w64-gmp                 6.1.0                         2    conda-forge
m2w64-libwinpthread-git   5.0.0.4634.697f757               2    conda-forge
markdown                  3.3.4              pyhd8ed1ab_0    conda-forge
markupsafe                2.0.1            py37hcc03f2d_0    conda-forge
matplotlib                3.3.0                    pypi_0    pypi
matplotlib-base           3.4.3            py37h4a79c79_0    conda-forge
mccabe                    0.6.1                      py_1    conda-forge
menuinst                  1.4.16           py37he774522_0
missingno                 0.4.2                    pypi_0    pypi
mistune                   0.8.4           py37hcc03f2d_1004    conda-forge
mkl                       2020.4             hb70f87d_311    conda-forge
mkl-service               2.3.0            py37h196d8e1_0
mkl_fft                   1.3.0            py37hda49f71_1    conda-forge
mkl_random                1.1.0            py37h675688f_0
mock                      4.0.3            py37h03978a9_1    conda-forge
more-itertools            8.8.0              pyhd8ed1ab_0    conda-forge
mpc                       1.1.0             h7edee0f_1009    conda-forge
mpfr                      4.1.0                h8d14728_0    conda-forge
mpir                      3.0.0             he025d50_1002    conda-forge
mpmath                    1.2.1              pyhd8ed1ab_0    conda-forge
msgpack-python            1.0.2            py37h8c56517_1    conda-forge
msys2-conda-epoch         20160418                      1    conda-forge
multipledispatch          0.6.0                      py_0    conda-forge
mypy                      0.782                    pypi_0    pypi
mypy-extensions           0.4.3                    pypi_0    pypi
navigator-updater         0.2.1                    py37_0  
nbclassic                 0.3.1              pyhd8ed1ab_1    conda-forge
nbclient                  0.5.3              pyhd8ed1ab_0    conda-forge
nbconvert                 6.1.0            py37h03978a9_0    conda-forge
nbformat                  5.1.3              pyhd8ed1ab_0    conda-forge
nest-asyncio              1.5.1              pyhd8ed1ab_0    conda-forge
networkx                  2.3                        py_0    conda-forge
ninja                     1.10.2               h5362a0b_0    conda-forge
nltk                      3.6.2              pyhd8ed1ab_0    conda-forge
nose                      1.3.7                   py_1006    conda-forge
notebook                  6.0.3                    py37_0
numba                     0.53.1           py37h9690626_1    conda-forge
numexpr                   2.7.3            py37h08fd248_0    conda-forge
numpy                     1.18.1                   pypi_0    pypi
numpy-base                1.18.5           py37hc3f5095_0
numpydoc                  1.1.0                      py_1    conda-forge
olefile                   0.46               pyh9f0ad1d_1    conda-forge
openjpeg                  2.4.0                hb211442_1    conda-forge
openpyxl                  3.0.7              pyhd8ed1ab_0    conda-forge
openssl                   1.1.1k               h8ffe710_0    conda-forge
packaging                 21.0               pyhd8ed1ab_0    conda-forge
pandas                    1.0.5                    pypi_0    pypi
pandas-profiling          2.8.0                    pypi_0    pypi
pandas-read-xml           0.0.8                    pypi_0    pypi
pandoc                    2.14.1               h8ffe710_0    conda-forge
pandocfilters             1.4.2                      py_1    conda-forge
panel                     0.12.1             pyhd8ed1ab_0    conda-forge
param                     1.11.1             pyh6c4a22f_0    conda-forge
paramiko                  2.7.2              pyh9f0ad1d_0    conda-forge
parso                     0.5.2                      py_0
partd                     1.2.0              pyhd8ed1ab_0    conda-forge
path                      16.2.0           py37h03978a9_0    conda-forge
path.py                   12.5.0                        0    conda-forge
pathlib2                  2.3.6            py37h03978a9_0    conda-forge
pathtools                 0.1.2                      py_1    conda-forge
patsy                     0.5.1                      py_0    conda-forge
pep8                      1.7.1                      py_0    conda-forge
pexpect                   4.8.0              pyh9f0ad1d_2    conda-forge
phik                      0.10.0                   pypi_0    pypi
pickleshare               0.7.5                   py_1003    conda-forge
pillow                    8.3.1            py37hd7d9ad0_0    conda-forge
pip                       21.2.4             pyhd8ed1ab_0    conda-forge
pkginfo                   1.7.1              pyhd8ed1ab_0    conda-forge
plotly                    4.9.0              pyh9f0ad1d_0    conda-forge
plotly-orca               1.3.1                         1    plotly
pluggy                    0.13.1           py37h03978a9_4    conda-forge
ply                       3.11                       py_1    conda-forge
pooch                     1.4.0              pyhd8ed1ab_0    conda-forge
powershell_shortcut       0.0.1                         3
prometheus_client         0.11.0             pyhd8ed1ab_0    conda-forge
prompt-toolkit            3.0.19             pyha770c72_0    conda-forge
prompt_toolkit            3.0.19               hd8ed1ab_0    conda-forge
psutil                    5.8.0            py37hcc03f2d_1    conda-forge
ptyprocess                0.7.0              pyhd3deb0d_0    conda-forge
py                        1.10.0             pyhd3deb0d_0    conda-forge
py-lief                   0.11.5           py37hf2a7229_0    conda-forge
pyarrow                   0.17.1                   pypi_0    pypi
pycodestyle               2.7.0              pyhd8ed1ab_0    conda-forge
pycosat                   0.6.3           py37hcc03f2d_1006    conda-forge
pycparser                 2.20               pyh9f0ad1d_2    conda-forge
pycrypto                  2.6.1           py37h0013d47_1005    conda-forge
pyct                      0.4.6                      py_0    conda-forge
pyct-core                 0.4.6                      py_0    conda-forge
pycurl                    7.43.0.6         py37h8b18dd5_1    conda-forge
pydocstyle                6.1.1              pyhd8ed1ab_0    conda-forge
pyerfa                    2.0.0            py37hebb4d16_0    conda-forge
pyflakes                  2.3.1              pyhd8ed1ab_0    conda-forge
pygments                  2.9.0              pyhd8ed1ab_0    conda-forge
pylint                    2.9.6              pyhd8ed1ab_0    conda-forge
pynacl                    1.4.0            py37ha54c9ec_2    conda-forge
pyodbc                    4.0.31           py37hf2a7229_0    conda-forge
pyopenssl                 20.0.1             pyhd8ed1ab_0    conda-forge
pyparsing                 2.4.7              pyh9f0ad1d_0    conda-forge
pyqt                      5.12.3           py37h03978a9_7    conda-forge
pyqt-impl                 5.12.3           py37hf2a7229_7    conda-forge
pyqt5-sip                 4.19.18          py37hf2a7229_7    conda-forge
pyqtchart                 5.12             py37hf2a7229_7    conda-forge
pyqtwebengine             5.12.1           py37hf2a7229_7    conda-forge
pyreadline                2.1             py37h03978a9_1003    conda-forge
pyrsistent                0.17.3           py37hcc03f2d_2    conda-forge
pysocks                   1.7.1            py37h03978a9_3    conda-forge
pytables                  3.6.1            py37hdc91d43_3    conda-forge
pytest                    6.2.4            py37h03978a9_0    conda-forge
python                    3.7.10          h7840368_100_cpython    conda-forge
python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
python-jsonrpc-server     0.4.0              pyh9f0ad1d_0    conda-forge
python-language-server    0.31.7                   py37_0
python-libarchive-c       3.1              py37h03978a9_0    conda-forge
python_abi                3.7                     2_cp37m    conda-forge
pytorch                   1.8.1           py3.7_cuda11.1_cudnn8_0    pytorch
pytz                      2020.4                   pypi_0    pypi
pyviz_comms               2.1.0              pyhd8ed1ab_0    conda-forge
pywavelets                1.1.1            py37hda49f71_3    conda-forge
pywin32                   301              py37hcc03f2d_0    conda-forge
pywin32-ctypes            0.2.0           py37h03978a9_1003    conda-forge
pywinpty                  1.1.3            py37h7f67f24_0    conda-forge
pyyaml                    5.4.1            py37hcc03f2d_0    conda-forge
pyzmq                     22.2.1           py37hcce574b_0    conda-forge
qdarkstyle                3.0.2              pyhd8ed1ab_0    conda-forge
qt                        5.12.9               h5909a2a_4    conda-forge
qtawesome                 1.0.3              pyhd8ed1ab_0    conda-forge
qtconsole                 5.1.1              pyhd8ed1ab_0    conda-forge
qtpy                      1.9.0                      py_0    conda-forge
regex                     2021.8.3         py37hcc03f2d_0    conda-forge
requests                  2.24.0                   pypi_0    pypi
requests-unixsocket       0.2.0                      py_0    conda-forge
retrying                  1.3.3                      py_2    conda-forge
rope                      0.19.0             pyhd8ed1ab_0    conda-forge
rtree                     0.9.7            py37h13cc57e_2    conda-forge
ruamel_yaml               0.15.80         py37hcc03f2d_1004    conda-forge
scikit-image              0.18.2           py37h90003fb_0    conda-forge
scikit-learn              0.24.2           py37hcabfae0_1    conda-forge
scipy                     1.7.1            py37hb6553fb_0    conda-forge
seaborn                   0.11.1               hd8ed1ab_1    conda-forge
seaborn-base              0.11.1             pyhd8ed1ab_1    conda-forge
send2trash                1.8.0              pyhd8ed1ab_0    conda-forge
setuptools                49.6.0           py37h03978a9_3    conda-forge
simplegeneric             0.8.1                      py_1    conda-forge
singledispatch            3.6.1              pyh44b312d_0    conda-forge
sip                       4.19.25          py37hf2a7229_0    conda-forge
six                       1.16.0             pyh6c4a22f_0    conda-forge
snappy                    1.1.8                ha925a31_3    conda-forge
sniffio                   1.2.0            py37h03978a9_1    conda-forge
snowballstemmer           2.1.0              pyhd8ed1ab_0    conda-forge
sodapy                    2.1.0                    pypi_0    pypi
sortedcollections         2.1.0              pyhd8ed1ab_0    conda-forge
sortedcontainers          2.4.0              pyhd8ed1ab_0    conda-forge
soupsieve                 2.0.1                      py_1    conda-forge
sphinx                    4.1.2              pyh6c4a22f_1    conda-forge
sphinxcontrib             1.0                      py37_1
sphinxcontrib-applehelp   1.0.2                      py_0    conda-forge
sphinxcontrib-devhelp     1.0.2                      py_0    conda-forge
sphinxcontrib-htmlhelp    2.0.0              pyhd8ed1ab_0    conda-forge
sphinxcontrib-jsmath      1.0.1                      py_0    conda-forge
sphinxcontrib-qthelp      1.0.3                      py_0    conda-forge
sphinxcontrib-serializinghtml 1.1.5              pyhd8ed1ab_0    conda-forge
sphinxcontrib-websupport  1.2.4              pyh9f0ad1d_0    conda-forge
spyder                    4.0.1                    py37_0
spyder-kernels            1.8.1                    py37_0
sqlalchemy                1.4.22           py37hcc03f2d_0    conda-forge
sqlite                    3.36.0               h8ffe710_0    conda-forge
statsmodels               0.12.2           py37hda49f71_0    conda-forge
sympy                     1.8              py37h03978a9_0    conda-forge
tangled-up-in-unicode     0.0.6                    pypi_0    pypi
tbb                       2020.2               h2d74725_4    conda-forge
tblib                     1.7.0              pyhd8ed1ab_0    conda-forge
terminado                 0.10.1           py37h03978a9_0    conda-forge
testpath                  0.5.0              pyhd8ed1ab_0    conda-forge
threadpoolctl             2.2.0              pyh8a188c0_0    conda-forge
tifffile                  2021.8.8           pyhd8ed1ab_0    conda-forge
tk                        8.6.10               h8ffe710_1    conda-forge
toml                      0.10.2             pyhd8ed1ab_0    conda-forge
toolz                     0.11.1                     py_0    conda-forge
torchaudio                0.8.1                      py37    pytorch
torchvision               0.2.2                      py_3    pytorch
tornado                   6.1              py37hcc03f2d_1    conda-forge
tqdm                      4.48.0                   pypi_0    pypi
traitlets                 5.0.5                      py_0    conda-forge
typed-ast                 1.4.1                    pypi_0    pypi
typing                    3.7.4.3                  pypi_0    pypi
typing-extensions         3.7.4.2                  pypi_0    pypi
typing_extensions         3.10.0.0           pyha770c72_0    conda-forge
tz                        0.2.2                    pypi_0    pypi
ucrt                      10.0.20348.0         h57928b3_0    conda-forge
ujson                     4.0.2            py37hf2a7229_0    conda-forge
unicodecsv                0.14.1                     py_1    conda-forge
urllib3                   1.26.6             pyhd8ed1ab_0    conda-forge
vc                        14.2                 hb210afc_5    conda-forge
vdom                      0.6                      pypi_0    pypi
virtualenv                20.0.29                  pypi_0    pypi
visions                   0.4.4                    pypi_0    pypi
vs2015_runtime            14.29.30037          h902a5da_5    conda-forge
watchdog                  2.1.3            py37h03978a9_0    conda-forge
wcwidth                   0.2.5              pyh9f0ad1d_2    conda-forge
webencodings              0.5.1                      py_1    conda-forge
websocket-client          0.57.0           py37h03978a9_4    conda-forge
werkzeug                  2.0.1              pyhd8ed1ab_0    conda-forge
wheel                     0.37.0             pyhd8ed1ab_1    conda-forge
widgetsnbextension        3.5.1            py37h03978a9_4    conda-forge
win_inet_pton             1.1.0            py37h03978a9_2    conda-forge
win_unicode_console       0.5             py37hc8dfbb8_1001    conda-forge
wincertstore              0.2             py37h03978a9_1006    conda-forge
winpty                    0.4.3                         4    conda-forge
wrapt                     1.12.1           py37hcc03f2d_3    conda-forge
xlrd                      2.0.1              pyhd8ed1ab_3    conda-forge
xlsxwriter                3.0.1              pyhd8ed1ab_0    conda-forge
xlwings                   0.24.7           py37h03978a9_0    conda-forge
xlwt                      1.3.0                      py_1    conda-forge
xmltodict                 0.12.0                     py_0    conda-forge
xz                        5.2.5                h62dcd97_1    conda-forge
yaml                      0.2.5                he774522_0    conda-forge
yapf                      0.31.0             pyhd8ed1ab_0    conda-forge
zeromq                    4.3.4                h0e60522_0    conda-forge
zfp                       0.5.5                h0e60522_5    conda-forge
zict                      2.0.0                      py_0    conda-forge
zipp                      3.5.0              pyhd8ed1ab_0    conda-forge
zlib                      1.2.11            h62dcd97_1010    conda-forge
zope                      1.0                      py37_1
zope.event                4.5.0              pyh9f0ad1d_0    conda-forge
zope.interface            5.4.0            py37hcc03f2d_0    conda-forge
zstd                      1.5.0                h6255e5f_0    conda-forge
PS C:\Users...>

**Comment by DavidKutu** [comment]
Another detail for the ipykernel 6 discussion, its only available on python 3.7 and up.

If I remember correctly, you have python 3.6. That would be why you can't use it.

**Comment by greazer** [comment]
Dupe of microsoft/vscode-jupyter-internal#190

**Comment by greazer** [comment]
I validated that I'm not running Python 3.6. Still not working. Here's the situation.... It's way messed up.

![image](https://user-images.githubusercontent.com/11685408/129796019-67d2bac6-4349-4132-a11a-5105ca28e53d.png)

I already allowed run-by-line to try and install ipykernel 6. After it was done, I still had ipykernel 5.5.5. I don't know how to get ipykernel 6 installed in this scenario. 

BUT I'm going to enter another bug related to the mess that this situation represents.
